CMakeLists.txt

```diff
@@ -2,6 +2,19 @@
 project(Halide)
 cmake_minimum_required(VERSION 3.2)
```

Comment:
Is 3.2 the minimum for CPack?

Reply:
The minimum for the CPack module is at least 3.0.0, if not earlier, considering it is part of the CMake suite.